pkg/cmd/application/create/app_create.go

```diff
@@ -73,6 +77,19 @@ NOTE: If parent directory is an existing git repository it will add a new applic
 		"",
 		"Tenant to configure for P2P",
 	)
+	appCreateCmd.Flags().StringVar(
```

Comment:
hmm, I though that this would be part of the software template, hardcoded there: i.e. corectl-args.yaml

Reply:
What is corectl-args.yaml?

app creation and template rendering should work in similar way in terms of template rendering from my perspective

Reply:
specific template requires specific number of arguments so it make sense to put arg descriptor file there and have opinionated place where it is